dependencies.yml

```diff
@@ -7,6 +7,8 @@
   version: 0.16.2
 - src: cloudalchemy.node-exporter
   version: 0.19.0
+- src: cloudalchemy.pushgateway
```

Comment:
We will also need to run/stop/start the service too, yes? Does it need supervisor?

Reply:
It will work with systemd.